spec/lib/bank_accounts/money_market_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require 'bank_accounts'
+
+describe BankAccounts::MoneyMarketAccount do
+
+ context "A money market account is a bank account" do
+   it "MoneyMarket class inherits from Account class" do
+     expect(BankAccounts::MoneyMarketAccount.superclass).to eq(BankAccounts::Account)
+   end
+ end
+
+ context "When creating a money market account" do
+  it "raises an error when instantiating a money market account with a balance below $10,000" do
+    expect{ BankAccounts::MoneyMarketAccount.new(0, 10) }.to raise_error(ArgumentError, "MINIMUM BALANCE $10000 REQUIRED")
+  end
+ end
+
+ context "When withdrawing from a money market account" do
+  let (:money_market) { BankAccounts::MoneyMarketAccount.new(0, 20000) }
+
+  it "returns an updated account balance after withdrawing" do
+    expect(money_market.withdraw(200)).to eq(19800)
+    expect(money_market.withdraw(4800)).to eq(15000)
```

Comment:
What do you see as the benefit of running two expectations here?
If you are testing that the value persists, perhaps run the withdrawal as a setup and then have the expectation be about the balance.